install/install.py

```diff
@@ -211,7 +215,11 @@ def install(cmd_arguments):
     # Installing pip and setting up virtualenv.
     # This requires distro specific dependencies to be installed properly.
     setup_pip()
-    setup_virtualenv()
+    if setup_virtualenv():
+        install_using_pip(owtf_pip)
+    else:
+        Colorizer.danger("Unable to setup virtualenv...")
```

Comment:
It would be more meaningful for debug to actually show the exception in that case. I am not against having a nice verbose message but how will we fix the issue if we don't know what happened?